Diplomatic.Core/Services/Serializers/TemplateConverter.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+
+namespace Diplomatic.Core
+{
+    public class TemplateConverter : CustomCreationConverter<Template>
+    {
+        public override Template Create(Type objectType)
+        {
+            throw new NotImplementedException();
+        }
+
+        public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+        {
+            var jObject = JObject.Load(reader);
+
+            var path = (string)jObject.Property("FilePath");
+            var fileStream = new FileStream(path);
+            Template target = new Template(fileStream);
```

Comment:
Should you use `var` instead of `Template` here? Is more consistent with the rest of the code here.

Reply:
And explicit type on line `18`